taskfile/taskfile_test.go

```diff
@@ -41,6 +43,16 @@ vars:
 				},
 			}},
 		},
+		{
+			yamlDeferredCmd,
+			&taskfile.Cmd{},
+			&taskfile.Cmd{Cmd: "echo 'test'", Defer: true},
+		},
+		{
+			yamlDeferredTask,
+			&taskfile.Cmd{},
+			&taskfile.Cmd{Task: "some_task", Defer: true},
+		},
```

Comment:
As explained in the previous comments, this second test should be removed as well.